docs/reference/modules/indices/recovery.asciidoc

```diff
@@ -21,6 +21,9 @@ peer recoveries:
     the cluster. Defaults to `40mb`.
 
 `indices.recovery.max_concurrent_file_chunks`::
+    Controls the number of file chunk requests can be sent in parallel per recovery.
```

Comment:
Controls the number of file chunk requests *that are* sent in parallel per *peer* recovery.